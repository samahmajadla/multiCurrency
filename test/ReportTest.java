import org.approvaltests.Approvals;
import org.junit.jupiter.api.Test;

import java.util.LinkedList;

class ReportTest {


    @Test
    public void generateReport(){

        LinkedList<Bond> bonds = new LinkedList<Bond>();
        bonds.add(new Bond("Integral", 300, new Dollar(2.5)));
        bonds.add(new Bond("Autonomic", 123, new Dollar(4.38)));
        Report report = new Report(bonds);
        Approvals.verify(report);


    }
}