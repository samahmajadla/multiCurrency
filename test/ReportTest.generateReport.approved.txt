Report{bonds=[Bond{instrument='Integral', shares=300, price=Dollar{amount=2.5}}, Bond{instrument='Autonomic', shares=123, price=Dollar{amount=4.38}}]}
